src/coreclr/ToolBox/superpmi/mcs/verbjitflags.cpp

```diff
@@ -0,0 +1,111 @@
+//
```

Comment:
question: looks like files in ToolBox/superpmi have their own license header, the standard one is:
```
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
```
is it expected?

Reply:
Thanks for catching that.

I think we need to redo all the license headers in these files. cc @BruceForstall